README.md

```diff
@@ -85,6 +85,8 @@ Installation instructions are available in [Docs/Book/Install.md](https://github
 - [Vernalis KNIME nodes](https://www.knime.com/book/vernalis-nodes-for-knime-trusted-extension)
 - [Erlwood KNIME nodes](https://www.knime.com/community/erlwood)
 - [AZOrange](https://github.com/AZcompTox/AZOrange)
+- [stk](https://github.com/lukasturcani/stk) ([docs](https://lukasturcani.github.io/stk/docs/build/html/), [paper](https://onlinelibrary.wiley.com/doi/10.1002/jcc.25377)) -
```

Comment:
I've been adding new projects at the top of the list so that the reader sees the newer stuff first. 
Please go ahead and move this to the top.